mindsdb/integrations/handlers/athena_handler/athena_handler.py

```diff
@@ -181,23 +181,18 @@ def get_columns(self, table_name: str) -> StatusResponse:
         return self.native_query(query)
 
     def _wait_for_query_to_complete(self, query_execution_id: str) -> str:
-        """
```

Comment:
Why the comment was removed?

Reply:
I accidentally removed the comment. Sorry!